NetKAN/CustomAsteroids-Pops-Stock-Stockalike.netkan

```diff
@@ -0,0 +1,42 @@
+{
+    "spec_version": 1,
+    "identifier": "CustomAsteroids-Pops-Stock-Stockalike",
+    "name": "Custom Asteroids (stockalike config)",
+    "abstract": "Emulates stock asteroids with Custom Asteroids groups.",
+    "$kref": "#/ckan/spacedock/210",
+    "$vref" : "#/ckan/ksp-avc",
+    "x_netkan_license_ok" : true,
+    "depends": [
+        {
+            "name": "CustomAsteroids",
+            "min_version": "v1.3.0",
+            "max_version": "v1.99.99",
+            "comment": "Requires intercept blocks and conditional spawning."
+        }
+    ],
+    "provides": [ "CustomAsteroids-Pops" ],
+    "conflicts": [
+        {
+            "name": "RealSolarSystem",
+            "comment": "The 6.4× and 10× Kerbol packs are compatible; how to allow them?"
```

Comment:
To clarify you mean this mod is compatible with `KSP-64k` (https://github.com/KSP-64k/64k)?